schematron/v2.1.0/tests/test_v2-1-0_L000_OpenStudio_Simulation.py

```diff
@@ -1,11 +1,12 @@
 import os
 
 from tools.validate_sch import validate_schematron
-from conftest import AssertFailureRolesMixin, SCH_DIR, exemplary_tree, remove_element
+from schematron.conftest import AssertFailureRolesMixin, exemplary_tree, remove_element
+from conftest import SCH_DIR
```

Comment:
seems like you could do:
```python
from schematron.conftest import AssertFailureRolesMixin, exemplary_tree, remove_element, v2_0_0_SCH_DIR
SCH_DIR = v2_0_0_SCH_DIR
```

Reply:
resolved by just importing from schematron/conftest.py